### ISSUE TITLE
Feature Request/Idea: 'Sticky' dataset file filtering

### ISSUE THREAD

**Issue by kuhlaid** [IQSS/dataverse#9917]
**What kind of user is the feature intended for?**
Everyone

**What inspired the request?**
Trying to filter a dataset but keep getting redirected to the top of the page and needing to scroll back down to the files.

**What existing behavior do you want changed?**
When a file filter is applied, for example if a user selects a different file type, file tag or even the file sorting option, this refreshes the page with the filtered content. The problem is that it sends the user back to the top of the page and thus loses focus on the file list, so the user if forced to scroll back down to the bottom of the page where the files are located. Ideally the file list would retain focus when filters selections changed by a user.

**Any brand new behavior do you want to add to Dataverse?**
One option would be to add an anchor tag at the dataset 'tabs' (Files, Metadata, Terms, Version) and when a filter is applied, the URL will include the anchor so that the page refresh keeps the files in focus (see screenshot below of the dataset tabs I am referring to). If the file list can be refreshed without redrawing the entire HTML page that would work as well but it depends on how you are choosing to handle content changes on a page.

![image](https://github.com/IQSS/dataverse/assets/41551429/ff91ef6c-f164-410f-a319-153c0d21e4b7)

**Comment by pdurbin** [comment]
@kuhlaid yeah, I agree this is pretty annoying! For now I'll say that we're working on this in the new frontend:

- https://github.com/IQSS/dataverse-frontend/pull/171

I'll try to remember to let you know when that PR has been merged and the code deployed to the beta server.

**Comment by DS-INRAE** [comment]
@pdurbin Should this issue have an SPA related tag so it is visible in order to address it then :) ?

**Comment by pdurbin** [comment]
@DS-INRAE well, above it looks like I was trying to say that I'd let you and @kuhlaid know when the pull request above has been merged and deployed to https://beta.dataverse.org

Can you reproduce the issue there on the beta server?

**Comment by kuhlaid** [comment]
It is still a problem on beta site. If you are on the dataset page under `Table View` and you choose a different `File Tag`, the page refreshes and takes you back to the top of the page. Maybe simply use anchor tags in the URL (e.g. #tabs). Thank you.

**Comment by pdurbin** [comment]
@kuhlaid thanks, would you be willing to create an issue at https://github.com/IQSS/dataverse-frontend about this?

**Comment by kuhlaid** [comment]
@pdurbin https://github.com/IQSS/dataverse-frontend/issues/552
